dynamic_rest/fields/fields.py

```diff
@@ -97,7 +99,9 @@ def __init__(
         self._serializer_class = serializer_class
         self.bound = False
         self.queryset = queryset
-        self.embed = embed
+        self.sideloading = sideloading
```

Comment:
Added this parameter to implementation, allowing consumers to force embedding/sideloading at a request level.